### DISCUSSION TITLE
[Enhancement] [table-v2] 虚拟表格希望可以增加一个类似与普通表格的合计功能，或者可以让某一行固定到表格底部

### DISCUSSION THREAD

**Discussion by ouweicong** [element-plus/element-plus#10249] [Q&A]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `table-v2`

## Description

虚拟表格希望可以增加一个类似与普通表格的合计功能，或者可以让某一行固定到表格底部

### Additional comments

_(empty)_

**Comment by chenxch** [comment]
[docs](https://element-plus.org/zh-CN/component/table-v2.html#%E8%87%AA%E5%AE%9A%E4%B9%89%E9%A1%B5%E8%84%9A)
组件具备自定义页脚的功能，你可以将页脚开发成你需要的功能哈
